pkg/util/ipvs/testing/fake.go

```diff
@@ -159,18 +177,19 @@ func (f *FakeIPVS) DeleteRealServer(serv *utilipvs.VirtualServer, dest *utilipvs
 		return fmt.Errorf("Failed to delete destination for service %v, service not found", key.String())
 	}
 	dests := f.Destinations[key]
-	var i int
-	for i = range dests {
-		if dests[i].Equal(dest) {
+	exist := false
+	for i := range dests {
+		if toRealServerKey(dests[i]).String() == toRealServerKey(dest).String() {
+			// Delete one element
+			f.Destinations[key] = append(f.Destinations[key][:i], f.Destinations[key][i+1:]...)
```

Comment:
Actually, there is no need to move this line here.

But it's ok if can make you happy :)

Reply:
So we can keep `i` inside the for cycle. ^_^